src/main/java/Diadoc/Api/sign/SignClient.java

```diff
@@ -163,6 +174,34 @@ public AsyncMethodResult dssSign(String boxId, DssSignRequest request, @Nullable
         }
     }
 
+    /**
+     * @deprecated Method is currently unavailable.
+     * Use {@link #dssSign(String, DssSignRequest, String)}
+     */
+    @Deprecated
+    public AsyncMethodResult dssSignV2(String boxId, DssSignRequestV2 request) throws DiadocSdkException {
+        if (boxId == null) {
+            throw new IllegalArgumentException("boxId");
+        }
+        if (request == null) {
+            throw new IllegalArgumentException("request");
+        }
+
+        try {
+            var url = new URIBuilder(diadocHttpClient.getBaseUrl())
+                    .setPath("V2/DssSign")
```

Comment:
Тоже для единообразия

```suggestion
                    .setPath("/V2/DssSign")
```

Reply:
Поправил